<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmachine</class>
 <widget class="QDialog" name="addmachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="vButton_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>382</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vButton_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>382</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>  Enter machines name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_vName"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="input_builtYear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="vRadio_built">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_built">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Was it built?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Type:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vRadio_notBuilt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yearBuilt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter year built:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
